dumpling/export/dump.go

```diff
@@ -349,9 +357,16 @@ func (d *Dumper) dumpDatabases(tctx *tcontext.Context, metaConn *sql.Conn, taskC
 				zap.String("table", table.Name))
 			meta, err := dumpTableMeta(conf, metaConn, dbName, table)
 			if err != nil {
-				return err
+				return errors.Trace(err)
+			}
+			// adjust table collation
+			if table.Type == TableTypeBase {
+				newCreateSQL, err := adjustTableCollation(parser1, meta.ShowCreateTable(), table.Collation)
+				if err != nil {
+					return errors.Trace(err)
+				}
+				meta.(*tableMeta).showCreateTable = newCreateSQL
 			}
```

Comment:
This part coule be put into L378~L382.
![image](https://user-images.githubusercontent.com/21104942/144352623-c66ef933-bb3c-40f1-bbd9-a690eef9de1b.png)